src/main/java/com/cvsgo/dto/user/UserResponseDto.java

```diff
@@ -0,0 +1,32 @@
+package com.cvsgo.dto.user;
+
+import com.cvsgo.dto.tag.TagResponseDto;
+import com.cvsgo.entity.User;
+import java.util.List;
+import lombok.Getter;
+
+@Getter
+public class UserResponseDto {
+
+    private final Long id;
+    private final String email;
+    private final String nickname;
+    private final String profileImageUrl;
+    private final List<TagResponseDto> tags;
+    private final long reviewLikeCount;
+
+    private UserResponseDto(User user, Long reviewLikeCount) {
```

Comment:
Long일 필요가 없을 것 같아요. 인자 넘겨주는 쪽에서도 long으로 넘겨주기도 하고 UserResponseDto 클래스 필드도 primitive 타입이라서요!
```suggestion
    private UserResponseDto(User user, long reviewLikeCount) {
```